### PR TITLE
Make timezn and locale config automatic, use apt instead of apt-get, …

### PR DESCRIPTION
**Pull request by mfucci**
…automatically generate env

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -45,40 +45,43 @@ cat <<EOF >$ROOTFS/root/rootfs_setup.sh
 # Add Android OS group
 addgroup --gid 1015 sdcard_rw
 addgroup --gid 3003 android_inet
-# Setup non-root user
-adduser --gecos "Non-root user" $NON_ROOT_USER
-usermod -a -G android_inet $NON_ROOT_USER
-usermod -a -G sudo $NON_ROOT_USER
-usermod -a -G sdcard_rw $NON_ROOT_USER
-# Allow _apt to use network and sdcard
+# Allow _apt to use network and root to use sdcard
```
Comment (mateusazis) [inline install.sh]: Typo: underscore before "apt".

File: host/linoid.sh [hunk 2]
```diff
@@ -85,6 +85,12 @@ if [[ "$1" == "--update_scripts" ]]; then
   exit 0
 fi
 
+if [[ "$1" == "--terminal" ]]; then
+  echo "Connecting to linoid in terminal mode"
+  $ADB shell -t su root chroot $LINOID_PATH /usr/bin/env -i HOME=/root USER="root" TERM="$TERM" PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin /bin/bash
```
Comment (mateusazis) [inline host/linoid.sh]: Any reason why $ADB_SUDO can't be called here?

File: host/linoid.sh [hunk 3]
```diff
@@ -122,7 +131,7 @@ if [[ "$1" == "--install" ]]; then
   echo "Done"
 
   echo "Launching install script"
-  $ADB_SUDO sh -c $SCRIPT_PATH_ANDROID/install.sh
+  $ADB shell -t su root $SCRIPT_PATH_ANDROID/install.sh
```
Comment (mateusazis) [inline host/linoid.sh]: Any reason why $ADB_SUDO can't be called here?

File: host/linoid.sh (hunk 2 above)
Comment (mfucci) [reply host/linoid.sh]: I need the "-t" option to make it interactive. Refactor to make it more clear

File: host/linoid.sh (hunk 3 above)
Comment (mfucci) [reply host/linoid.sh]: Refactored

File: install.sh (hunk 1 above)
Comment (mfucci) [reply install.sh]: Updated comment to make it more clear

**Pull request merged**